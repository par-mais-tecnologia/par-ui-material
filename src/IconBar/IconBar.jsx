import React, { PureComponent } from 'react'
import Grid from '../Grid'

export default class IconBar extends PureComponent {
  render () {
    return (
      <Grid container justify='center' alignItems='center' className='mt2'>
        <a className='no-underline mh1 mh4-ns'
          href='https://www.facebook.com/parmais'
          target='_blank'
          rel='noopener noreferrer'>
          <img
            border='0'
            alt='Facebook'
            src='https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/facebook_contact.svg'
            width='50'
            height='50' />
        </a>
        <a className='no-underline mh1 mh4-ns'
          href='https://www.instagram.com/parmais'
          target='_blank'
          rel='noopener noreferrer'>
          <img
            border='0'
            alt='Instagram'
            src='https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/instagram_contact.svg'
            width='50'
            height='50' />
        </a>
        <a className='no-underline mh1 mh4-ns'
          href='https://www.youtube.com/parmais'
          target='_blank'
          rel='noopener noreferrer'>
          <img
            border='0'
            alt='Youtube'
            src='https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/youtube_contact.svg'
            width='50'
            height='50' />
        </a>
        <a className='no-underline mh1 mh4-ns'
          href='https://br.linkedin.com/company/par-mais'
          target='_blank'
          rel='noopener noreferrer'>
          <img
            border='0'
            alt='Linkedin'
            src='https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/linkedin_contact.svg'
            width='50'
            height='50' />
        </a>
      </Grid>
    )
  }
}
