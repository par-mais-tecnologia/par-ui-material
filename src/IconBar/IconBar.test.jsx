import React from 'react'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import IconBar from './'

configure({ adapter: new Adapter() })

export const test = describe('IconBar', () => {
  it('shallow IconBar', () => {
    const wrapper = shallow(
      <IconBar />
    )
    expect(wrapper).toBeTruthy()
  })

  it('shallow IconBar match Snapshot', () => {
    const wrapper = shallow(
      <IconBar />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
