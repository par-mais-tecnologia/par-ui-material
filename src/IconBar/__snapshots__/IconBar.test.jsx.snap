// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconBar shallow IconBar match Snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <IconBar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "alignItems": "center",
      "children": Array [
        <a
          className="no-underline mh1 mh4-ns"
          href="https://www.facebook.com/parmais"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="Facebook"
            border="0"
            height="50"
            src="https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/facebook_contact.svg"
            width="50"
          />
        </a>,
        <a
          className="no-underline mh1 mh4-ns"
          href="https://www.instagram.com/parmais"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="Instagram"
            border="0"
            height="50"
            src="https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/instagram_contact.svg"
            width="50"
          />
        </a>,
        <a
          className="no-underline mh1 mh4-ns"
          href="https://www.youtube.com/parmais"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="Youtube"
            border="0"
            height="50"
            src="https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/youtube_contact.svg"
            width="50"
          />
        </a>,
        <a
          className="no-underline mh1 mh4-ns"
          href="https://br.linkedin.com/company/par-mais"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="Linkedin"
            border="0"
            height="50"
            src="https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/linkedin_contact.svg"
            width="50"
          />
        </a>,
      ],
      "className": "mt2",
      "container": true,
      "justify": "center",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <img
            alt="Facebook"
            border="0"
            height="50"
            src="https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/facebook_contact.svg"
            width="50"
          />,
          "className": "no-underline mh1 mh4-ns",
          "href": "https://www.facebook.com/parmais",
          "rel": "noopener noreferrer",
          "target": "_blank",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "Facebook",
            "border": "0",
            "height": "50",
            "src": "https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/facebook_contact.svg",
            "width": "50",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <img
            alt="Instagram"
            border="0"
            height="50"
            src="https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/instagram_contact.svg"
            width="50"
          />,
          "className": "no-underline mh1 mh4-ns",
          "href": "https://www.instagram.com/parmais",
          "rel": "noopener noreferrer",
          "target": "_blank",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "Instagram",
            "border": "0",
            "height": "50",
            "src": "https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/instagram_contact.svg",
            "width": "50",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <img
            alt="Youtube"
            border="0"
            height="50"
            src="https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/youtube_contact.svg"
            width="50"
          />,
          "className": "no-underline mh1 mh4-ns",
          "href": "https://www.youtube.com/parmais",
          "rel": "noopener noreferrer",
          "target": "_blank",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "Youtube",
            "border": "0",
            "height": "50",
            "src": "https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/youtube_contact.svg",
            "width": "50",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <img
            alt="Linkedin"
            border="0"
            height="50"
            src="https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/linkedin_contact.svg"
            width="50"
          />,
          "className": "no-underline mh1 mh4-ns",
          "href": "https://br.linkedin.com/company/par-mais",
          "rel": "noopener noreferrer",
          "target": "_blank",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "Linkedin",
            "border": "0",
            "height": "50",
            "src": "https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/linkedin_contact.svg",
            "width": "50",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": "a",
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "alignItems": "center",
        "children": Array [
          <a
            className="no-underline mh1 mh4-ns"
            href="https://www.facebook.com/parmais"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Facebook"
              border="0"
              height="50"
              src="https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/facebook_contact.svg"
              width="50"
            />
          </a>,
          <a
            className="no-underline mh1 mh4-ns"
            href="https://www.instagram.com/parmais"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Instagram"
              border="0"
              height="50"
              src="https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/instagram_contact.svg"
              width="50"
            />
          </a>,
          <a
            className="no-underline mh1 mh4-ns"
            href="https://www.youtube.com/parmais"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Youtube"
              border="0"
              height="50"
              src="https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/youtube_contact.svg"
              width="50"
            />
          </a>,
          <a
            className="no-underline mh1 mh4-ns"
            href="https://br.linkedin.com/company/par-mais"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin"
              border="0"
              height="50"
              src="https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/linkedin_contact.svg"
              width="50"
            />
          </a>,
        ],
        "className": "mt2",
        "container": true,
        "justify": "center",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              alt="Facebook"
              border="0"
              height="50"
              src="https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/facebook_contact.svg"
              width="50"
            />,
            "className": "no-underline mh1 mh4-ns",
            "href": "https://www.facebook.com/parmais",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "Facebook",
              "border": "0",
              "height": "50",
              "src": "https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/facebook_contact.svg",
              "width": "50",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              alt="Instagram"
              border="0"
              height="50"
              src="https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/instagram_contact.svg"
              width="50"
            />,
            "className": "no-underline mh1 mh4-ns",
            "href": "https://www.instagram.com/parmais",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "Instagram",
              "border": "0",
              "height": "50",
              "src": "https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/instagram_contact.svg",
              "width": "50",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              alt="Youtube"
              border="0"
              height="50"
              src="https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/youtube_contact.svg"
              width="50"
            />,
            "className": "no-underline mh1 mh4-ns",
            "href": "https://www.youtube.com/parmais",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "Youtube",
              "border": "0",
              "height": "50",
              "src": "https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/youtube_contact.svg",
              "width": "50",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              alt="Linkedin"
              border="0"
              height="50"
              src="https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/linkedin_contact.svg"
              width="50"
            />,
            "className": "no-underline mh1 mh4-ns",
            "href": "https://br.linkedin.com/company/par-mais",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "Linkedin",
              "border": "0",
              "height": "50",
              "src": "https://s3-sa-east-1.amazonaws.com/static.parmais.com.br/images/linkedin_contact.svg",
              "width": "50",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "a",
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
